sdk/src/settings/builder.rs

```diff
@@ -467,10 +467,25 @@ pub struct BuilderSettings {
     /// See more information on the difference between created vs gathered assertions in the spec here:
     /// <https://spec.c2pa.org/specifications/specifications/2.2/specs/C2PA_Specification.html#_fields>
     pub created_assertion_labels: Option<Vec<String>>,
-
     /// Whether to generate a C2PA archive (instead of zip) when writing the manifest builder.
     /// This will eventually become the default behavior.
     pub generate_c2pa_archive: Option<bool>,
+    /// Whether to auto-generate a [`TimeStamp`] assertion for parent of the update manifest if one does not
+    /// exist already.
+    ///
+    /// Useful when a manifest was signed offline and you want to attach a trusted timestamp to it later.
+    ///
+    /// Note that for this setting to take effect, a time authority URL must be set in the
+    /// [`Signer::time_authority_url`]. If the signer is acquired from settings via [`Settings::signer`],
+    /// the URL can be set in [`SignerSettings`].
+    ///
+    /// The default value is false.
+    ///
+    /// [`TimeStamp`]: crate::assertions::TimeStamp
+    /// [`Signer::time_authority_url`]: crate::Signer::time_authority_url
+    /// [`Settings::signer`]: crate::settings::signer
+    /// [`SignerSettings`]: crate::settings::signer::SignerSettings
+    pub update_manifest_timestamp_assertion: bool,
```

Comment:
You can add a timestamp to regular manifests too, not just update manifests.  Those are separate issues.

Reply:
Good point, this specific example was discussed as a common use case, but I don't see a problem in generalizing it to any parent ingredient. I'll change it to `add_timestamp_assertion_to_parent`.